docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -558,6 +561,70 @@ The default value is `fail`, which causes the migrate procedure to throw an
 exception if subdirectories are found. Set the value to `true` to migrate
 nested directories, or `false` to ignore them.
 
+(iceberg-add-files)=
+#### Add files
+
+The connector can add files from tables or locations if
+`iceberg.add-files-procedure.enabled` is set to `true` for the catalog.
+
+Use the procedure `system.add_files` to add existing files from the Hive table or 
+specified locations. The data files must be the Parquet, ORC, or Avro file format.
+
+:::{warning}
+The procedure does not check if files are already present in the target table.
+:::
+
+The procedure must be called for a specific catalog `example` with the
+relevant schema and table names supplied with the required parameters
+`target_schema_name` and `target_table_name`:
+
+```sql
+CALL example.system.add_files(
+    target_schema_name => 'testdb',
+    target_table_name => 'iceberg_customer_orders',
+    source_schema_name => 'testdb',
+    source_table_name => 'hive_customer_orders')
+```
+
+You can provide a `partition_filter` argument to add files from specified partitions:
```

Comment:
This sentence assumes that it's optional but it's actually required in case of partition tables, otherwise first example fill fail.

I would assume that if no filter is specified then all files will be copied as in example above.
Specifying all partitions may also lead to complicated syntax in case table with many partitions.